omegaconf/omegaconf.py

```diff
@@ -183,7 +183,7 @@ def create(
     @staticmethod
     @overload
     def create(
-        obj: Union[Dict[str, Any], None] = None,
+        obj: Optional[Dict[Any, Any]] = None,
```

Comment:
Why did you end up using `Any` here instead of `DictKeyType`?

Reply:
Ah, never mind, just saw the answer: https://github.com/omry/omegaconf/pull/454#pullrequestreview-550948711